packages/backend/src/util/tree/fetchJsonLines.ts

```diff
@@ -0,0 +1,55 @@
+import { ReadableStream } from 'stream/web';
+
+export async function* fetchJsonLines<T = any>(
```

Comment:
what are you using this for?

also why is it under "util/tree". There is nothing "tree"-ish about this.

Reply:
In the Next handler tests, to read the stream response. Will move to tests